tests/samples/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10.11-bullseye
```

Comment:
Can we have somewhere a docs that explains how to run these? Either in the README either BENCHMARKING.md for instance